xmind/tests/test_sheet_element.py

```diff
@@ -0,0 +1,402 @@
+import logging
+from xmind.core.sheet import SheetElement
+from xmind.tests import base
+from unittest.mock import patch, Mock, MagicMock
+from xmind.core.const import *
+
+
+
+class TestSheetElement(base.Base):
+    """Test class for SheetElement class"""
+
+    def getLogger(self):
+        if not getattr(self, '_logger', None):
+            self._logger = logging.getLogger('sheetElement')
+        return self._logger
+
+    def setUp(self):
```

Comment:
```_get_root_topic``` has to be in ```setUp``` to avoid duplicates in your code

Reply:
I see, but how I will test _get_root_topic  method ?

Reply:
for that case you can just stop patch by calling ```self. _remove_patched_function()``` by passing name of the patch